tests/v2/conftest.py

```diff
@@ -1,5 +1,5 @@
-import pathlib
 import pytest
+import pathlib
```

Comment:
Should we use isort or something else automated?

Reply:
yes, I would love this. i think ruff format takes a parameter that does what we want. I will add that to this PR unless there any objections.

Reply:
instead of adding isort-style linting to this PR, it looks like we are going to do that in a separate effort.